README.rst

```diff
@@ -399,6 +399,10 @@ Sphinx deprecation warnings. These warnings match the following regular expressi
 `RemovedInSphinx\\d+Warning`. Using this flag results in the same behavior as adding this
 regex to the configuration file as value for the `exclude` key for the sphinx checker.
 
+Store All Counted Warnings
+--------------------------
+
+Use `-o, --output <file_path>` to let the plugin write all counted warnings/failures as strings to a file.
```

Comment:
Can we add example of this output file maybe? It will be helpful when the output changes or new output file format is added (so maybe we use `--format` flag for that -> and we default it to string at begining)

Reply:
I'll add examples of counted warnings/failures for the different checker types.

Which values for this `--format` option did you have in mind? I don't understand what you mean.

Reply:
Now it is a file
```
5
4
```

But what I would assume we could have json output (something special again), XML (JUnit) output, etc?

Reply:
When `<file_path>` ends with `.xml`, the plugin will know that is should produce JUnit output when it is supported.

Reply:
Done in 9da54fafd3d0b4fd537ae13db7bbb8957de4479a